tgui/packages/tgui/interfaces/ChemDispenser/types.ts

```diff
@@ -8,6 +13,8 @@ export type Data = {
   beakerCurrentVolume: number | null;
   beakerMaxVolume: number | null;
   chemicals: reagent[];
+  recipes: { [key: string]: Recipe[] };
```

Comment:
Wouldn't Record<string, Recipe[]> be easier?

Reply:
I guess it would be, I'm just used to doing it this way because it works